snafu/scale_wrapper/trigger_scale.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import os
+import subprocess
+import json
+import socket
+import dateutil.parser
+import logging
+import yaml
+import time
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+
+logger = logging.getLogger("snafu")
+
+
+class Trigger_scale():
+    def __init__(self, args):
+        self.uuid = args.uuid
+        self.user = args.user
+        self.scale = args.scale
+        self.cluster_name = args.cluster_name
+        if args.results :
```

Comment:
how would init come with the results?

Reply:
It's a carryover from the workload I copied the base skel from. I'll clean it up